javav2/example_code/sns/Readme.md

```diff
@@ -1,6 +1,52 @@
 # Amazon SNS Java code examples
```

Comment:
Amazon SNS code examples for the SDK for Java

Reply:
fixed